src/firewheel_repo_linux/linux/INSTALL/tasks.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Check if Linux Base Objects is already installed
+  ansible.builtin.stat:
+    path: "{{ install_flag }}"
+  register: flag_check
+
+- name: Fail if already installed
+  ansible.builtin.fail:
+    msg: "linux.base_objects is already installed!"
+  when: flag_check.stat.exists
+
+- name: Create VM resources directory
+  ansible.builtin.file:
+    path: "{{ vm_resources_dir }}"
+    state: directory
+
+- name: Download and verify files
+  ansible.builtin.get_url:
+    url: "{{ item.url }}"
+    dest: "{{ vm_resources_dir }}/{{ item.parent }}/{{ item.dest }}"
+    checksum: "sha256:{{ item.sha256 }}"
+  loop: "{{ files }}"
```

Comment:
I'm a bit confused on how this will work. It looks like the "files" are actually defined in `profiles`? but there isn't a `parent` or `dest` associated? Also, how will this generate a new SSH key?